src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -275,6 +275,12 @@ public string Author
             }
         }
 
+        public string Description
+        {
+            get;
+            set;
+        }
```

Comment:
Let's make sure we read and write this to the dyn/dyf file. Let's also make sure that there are tests for this. Please add a `RaisePropertyChanged` event.